python/ray/tune/suggest/util.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from typing import Dict, Optional, List
+
+from ray.tune.experiment import Experiment
+
+logger = logging.getLogger(__name__)
+
+
+def set_search_properties_backwards_compatible(
+        set_search_properties_func, metric: Optional[str], mode: Optional[str],
+        config: Dict, experiment: Optional[Experiment]) -> bool:
+    try:
```

Comment:
Can we add a short docstring explaining what this is for? One, two sentences

Reply:
Added :)